server/app/middlewares/auth.js

```diff
@@ -0,0 +1,24 @@
+import validator from 'email-validator';
```

Comment:
You cannot use this validator. You have to write your own custom email validator.

Reply:
done